warehouse/migrations/versions/b74a66a8f312_cascade_release_deletion_to_release_.py

```diff
@@ -24,8 +24,9 @@
 
 
 def upgrade():
-    op.drop_constraint(
-        "release_classifiers_name_fkey", "release_classifiers", type_="foreignkey"
+    op.execute(
+        "ALTER TABLE release_classifiers "
+        "DROP CONSTRAINT IF EXISTS release_classifiers_name_fkey"
```

Comment:
I wrote up https://github.com/sqlalchemy/sqlalchemy/issues/8141 to capture the use case here, since it would be nice to not have to write our own SQL 😁